packages/next/next-server/server/next-server.ts

```
@@ -151,8 +168,14 @@ export default class Server {
       })
     }
 
-    const routes = this.generateRoutes()
-    this.router = new Router(routes)
+    this.customRoutes = { redirects: [], rewrites: [] }
+    if (!dev) {
+      try {
+        this.customRoutes = require(join(this.distDir, ROUTES_MANIFEST))
+      } catch (_) {}
```

This error should never be swallowed, not sure why we'd do this.